src/mongoose_local_delivery.erl

```diff
@@ -40,7 +40,7 @@ do_route(OrigFrom, OrigTo, OrigAcc, OrigPacket, Handler) ->
                     mongoose_packet_handler:process(Handler, Acc1, From, To, Packet);
                 drop ->
                     mongoose_hooks:xmpp_stanza_dropped(mongoose_acc:host_type(Acc0), OrigFrom,
-                                                       OrigTo, OrigPacket),
+                                                       OrigTo, OrigPacket, Acc0),
```

Comment:
Do we need put here `Acc0` instead `OrigAcc`?

Reply:
Looks like yes, cuz `host_type` is get from `Acc0`.

Reply:
`Acc0` is correct here because it contains the host type of the recipient.